src/ovp_pipeline/auto_evergreen_extractor.py

```
@@ -612,6 +612,46 @@ def extract_concepts(
 
         return self._parse_v2_response(result_text, file_path)
 
+    def _build_router_llm(self) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_build_router_llm`

(ANN401)